bee-ternary/src/lib.rs

```diff
@@ -68,6 +68,8 @@ pub mod t3b1;
 pub mod t4b1;
 /// The [`T5B1`] and [`T5B1Buf`] encodings.
 pub mod t5b1;
+/// Utility functions for encoding and decodining T6B1 encoding.
+pub mod t6b1;
```

Comment:
I am not sure this is the appropriate naming for this encoding module. It makes it seems like we could also encode 6 trits in 1 byte, which is not possible. Moreover, this encoding is - in principle - the inverse of the t5b1 encoding with the difference that now we need to be able to encode 256 different values which implies at least 6 trits. Therefore, this encoding should be named `b1t6` to indicate the direction: "from *any* b1 to t6", while `t5b1` is "from *any* t5 to b1".

Reply:
While I kinda agree with what you are saying, this is the name given by the RFC https://github.com/iotaledger/protocol-rfcs/blob/master/text/0015-binary-to-ternary-encoding/0015-binary-to-ternary-encoding.md. @zesterer Maybe put a link to this RFC somewhere for clarity.

Reply:
The RFC actually confirms my comment :smile:

Reply:
I misread your initial comment. As said, I don't think reversing the name changes anything because it is still working only one side. So it's only a matter of if you prefer to read it "To 6 trits from 1 byte" (t6b1) or "From 1 byte to 6 trits" b1t6, but they both are exactly the same. For the sake of consistency I do prefer t6b1 but you guys choose.

Reply:
I've flipped the naming scheme to B1T6 for now. If other ideas exist it would be nice to hear them.